CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-cmake_minimum_required(VERSION 3.12) # Don't bump this version for no reason
+# 3.13 is required for target_link_libraries. 
```

Comment:
target_link_libraries was available before CMake 3.13, although its behavior has changed slightly with 3.13. Have you confirmed that 3.12 actually doesn't work?

Reply:
Yes. I have. That was the first thing I ran, and then stuck with 3.13.x once it started working.

Reply:
P.S. Issue with 3.12.4 

![Screenshot-CMake-Issues](https://github.com/ggerganov/llama.cpp/assets/195178/476121e2-4daa-4932-a1b0-7def5df270f4)